honeybadger.go

```diff
@@ -11,11 +15,30 @@ type HoneyBadger struct {
 
 	// current epoch
 	epoch int
+
+	// b is maximum number of transactions in each epoch
+	b int
+
+	// n is number of node at network
+	n int
+}
+
+func NewHoneyBadger(batch int, nodes int) *HoneyBadger {
+	if batch < nodes {
```

Comment:
Using `batch` in returning `HoneyBadger` can be confused. Because `b` could be `batch` or `nodes`. 

In good code view, I think making a new variable like `maxTx` would be better.

For example,
```go
func NewHoneyBadger(batch int, nodes int) *HoneyBadger {
	maxTx := batch

	if batch < nodes {
		maxTx = nodes
	}

 	return &HoneyBadger{
		que:   NewQueue(),
		epoch: 0,
		b:     maxTx,
		n:     nodes,
	}
}	
```